nautobot/core/views/generic.py

```diff
@@ -1149,6 +1149,10 @@ def get_required_permission(self):
     def post(self, request):
         logger = logging.getLogger("nautobot.views.BulkRenameView")
 
+        if not request.POST.getlist("pk"):
```

Comment:
Something I noticed in looking at a different issue with bulk-edit views is that we silently filter out and discard any objects that are in `pk` list but aren't editable by the current user (`selected_objects = self.queryset.filter(pk__in=form.initial["pk"])`). So there is a case where the provided list is non-empty, but the actual operation will be a no-op because no valid objects were included. Might want to handle that case?